about.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link
+      href="http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="container-md d-flex flex-column align-items-center my-5">
+      <h1
+        class="about-header display-2 fw-bolder text-uppercase text-center my-2"
+      >
+        "Jah Malet Fetari" International Chess Tournament 2023
+      </h1>
+      <div class="sub-heading text-center my-2">
```

Comment:
- [ ] Kindly 😃 try to wrap the content here in the section tag that's for the semantic reason you can read more about the semantics [Here](https://www.w3schools.com/html/html5_semantic_elements.asp)😊⤵️